test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+pre-commit==1.17.0
```

Comment:
Now, the runtime requirements are also needed. Just don't copy-paste things. You can just refer to the other file like this:
```suggestion
-r requirements.txt
pre-commit==1.17.0
```

Reply:
This is still unsolved.